spec.bs

```diff
@@ -1830,70 +1845,66 @@ null |winningComponentConfig|:
      [=leading bid info/component seller reporting result=] to |reportingResult|.
   1. Otherwise, set |leadingBidInfo|'s [=leading bid info/seller reporting result=] to
      |reportingResult|.
+  1. [=map/Remove=] |browserSignals|["`desirability`"].
   1. [=map/Remove=] |browserSignals|["`dataVersion`"].

```

Comment:
Most of these removed fields are the ones that aren't in [ReportingBrowserSignals](https://pr-preview.s3.amazonaws.com/qingxinwu/turtledove/pull/678.html#dictdef-reportingbrowsersignals), but `dataVersion` is in there. Mention in your note why it gets removed too?